### ISSUE TITLE
Fatal Error while running ./vendor/bin/phpunit

### ISSUE THREAD

**Issue by junaidtk** [sebastianbergmann/phpunit#5036]
I am running the phpunit testing for my WordPress plugin. But when running phpunit test I get an error.
(./vendor/bin/phpunit). 

I have checked the issue many times,  but I didn't find a proper solution to solve my issue.

While running the ./vendor/bin/phpunit command I am getting the below issue.

`Installing...
Running as single site... To run multisite, use -c tests/phpunit/multisite.xml
Not running ajax tests. To execute these, use --group ajax.
Not running ms-files tests. To execute these, use --group ms-files.
Not running external-http tests. To execute these, use --group external-http.

Fatal error: Uncaught ParseError: syntax error, unexpected 's' (T_STRING), expecting function (T_FUNCTION) or const (T_CONST) in /Users/junaid/work/www/php-unittest/wp-content/plugins/wp-meta-verify/vendor/phpunit/phpunit/src/TextUI/Command.php on line 98

PHPUnit\TextUI\RuntimeException: syntax error, unexpected 's' (T_STRING), expecting function (T_FUNCTION) or const (T_CONST) in /Users/junaid/work/www/php-unittest/wp-content/plugins/wp-meta-verify/vendor/phpunit/phpunit/src/TextUI/Command.php on line 98

Call Stack:
    0.0063     415928   1. {main}() /Users/junaid/work/www/php-unittest/wp-content/plugins/wp-meta-verify/vendor/bin/phpunit:0
    0.0191     429560   2. include('phpvfscomposer:///Users/junaid/work/www/php-unittest/wp-content/plugins/wp-meta-verify/vendor/phpunit/phpunit/phpunit') /Users/junaid/work/www/php-unittest/wp-content/plugins/wp-meta-verify/vendor/bin/phpunit:115
    0.0364    1253208   3. PHPUnit\TextUI\Command::main() phpvfscomposer:///Users/junaid/work/www/php-unittest/wp-content/plugins/wp-meta-verify/vendor/phpunit/phpunit/phpunit:97
`
My PHP version is PHP 7.4.12.

./vendor/bin/phpunit --version is given below.
PHPUnit 9.5.23

**Comment by sebastianbergmann** [comment]
Thank you for your report.

Please provide a **minimal**, **self-contained**, reproducing test case that shows the problem you are reporting.

Without such a minimal, self-contained, reproducing test case I will not be able to investigate this issue.

**Comment by junaidtk** [comment]
Thank you for your fast reply. 

I am running the phpunit test for my WordPress plugin. So I am starting my unit testing. 

I have followed the below articles to configure my test.

https://sarathlal.com/unit-testing-wordpress-plugin-using-phpunit/

I followed exactly the same as in the article. After step 6, I am getting the above issue.

I am using macOS Big Sur. If any specific data required about the issue and system. I am happy to share.
